cmd/root.go

```diff
@@ -249,11 +258,35 @@ func parseConfig(cmd *cobra.Command, conf *proxy.Config, args []string) error {
 	return nil
 }
 
+func configureTelemetry(projectID, metricPrefix string) (func(), error) {
```

Comment:
Can we make more clear this is for Stackdriver (or Cloud Trace / Metrics)?

Reply:
Done.